.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: ExceptionPage CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: "*"
+
+jobs:
+  check_format:
+    strategy:
+      fail-fast: false
+      matrix:
+        crystal_version:
+          - 0.36.1
+          - 1.0.0
+        experimental:
+          - false
```

Comment:
I haven't tried it out yet, but we can add in nightly as experimental in the future if we wanted.